frontend/src/relay/auth.tsx

```diff
@@ -77,7 +77,14 @@ export const ExternalLink: React.FC<ExternalLinkProps> = ({
         <input type="hidden" name="target" value={target.toString()} />
         <input type="hidden" name="jwt" />
 
-        <button className={className}>{children}</button>
+        <button
+            className={className}
+            css={{
+                backgroundColor: "transparent",
+                border: "none",
+                minWidth: "100%",
+            }}
```

Comment:
If this is supposed to be here you'd need to adjust the the `ExternalLink` on the "Video details" page as it spans the whole container now. Or is that intentional?
Also, `min-width: 100%` seems a little weird, wouldn't `width: 100%` do the same?

<img width="1165" alt="Bildschirmfoto 2023-07-12 um 17 31 10" src="https://github.com/elan-ev/tobira/assets/94838646/7dd54683-caea-45e5-bf4a-65a386f8d182">

Reply:
Oh woops, good catch

Reply:
Just removing this one `min-width` line seems to do the trick. Doesn't seem to be needed. The button will get `min-width: 160px` anyway, like all other items.